.travis.yml

```diff
@@ -32,38 +32,70 @@ jobs:
       #if: tag =~ /^release.*$/
       language: shell
       services: docker
+      install:
+        # set up awscli packages
+        - python3 -m pip install --user awscli

```

Comment:
This does install AWS CLI as Python module. Frankly, this is the easiest way to do it (as per Travis docs). I've added a specific v1 version (`1.20.9`) to be installed.

Side note: one can't install AWS CLI v2 via pip because it is not available in PyPi (some arbitrary AWS decision).